low-level/files/opt/seagate/sspl/setup/resource_map/server.py

```diff
@@ -269,6 +273,68 @@ def prepare_mem_json(self, status, description):
         }
         return memory_dict
 
+    def get_nw_ports_info(self):
+        """Return the Network ports information."""
+        network_cable_data = []
+        io_counters = psutil.net_io_counters(pernic=True)
+
+        for interface, addrs in psutil.net_if_addrs().items():
+            nic_info = self.get_health_template(interface, False)
+
+            specifics = {}
+            for addr in addrs:
+                if addr.family == AF_INET:
+                    specifics["ipV4"] = addr.address
+
+            if interface in io_counters:
+                io_info = io_counters[interface]
+                specifics = {
+                    "networkErrors": io_info.errin + io_info.errout,
+                    "droppedPacketsIn": io_info.dropin,
+                    "droppedPacketsOut": io_info.dropout,
+                    "packetsIn": io_info.packets_recv,
+                    "packetsOut": io_info.packets_sent,
+                    "trafficIn": io_info.bytes_recv,
+                    "trafficOut": io_info.bytes_sent
+                }
+
+            nw_status, nw_cable_conn_status = \
+                self.get_nw_status(interface)
```

Comment:
Is this exceeds 80 chars in line? If not, don't wrap.

Reply:
now with added arg it does, no kept is it wrapped.